data/Localization/german_(germany)/global.ini

```diff
@@ -27877,7 +27888,6 @@ PU_GENOUTLAW2_M_CB_Relaxed_NoticeBulletImpactsClose_ThreatHigh_High_IG_002_ShitT
 PU_GENOUTLAW2_M_CB_Relaxed_NoticeBulletImpactsClose_ThreatHigh_High_IG_003_SomeBastardsAfter=Irgendein Mistkerl ist hinter mir her!
 PU_GENOUTLAW2_M_CB_Relaxed_NoticeCorpse_ThreatMed_IG_001_FoundAFrickin=Ich habe eine verdammte Leiche gefunden.
 PU_GENOUTLAW2_M_CB_Relaxed_NoticeCorpse_ThreatMed_IG_002_WonderWhatThis=Ich frage mich, was er falsch gemacht hat.\n
-PU_Relaxed_NoticeCorpse_ThreatMed_IG_003_WhoKilledEm=Wer hat sie getötet? War das einer von euch?
 PU_GENOUTLAW2_M_CB_Relaxed_NoticeCorpse_ThreatMed_IG_003_WhoKilledEm=Wer hat sie getötet? War das einer von euch?
 PU_GENOUTLAW2_M_CB_Relaxed_NoticeFriendlyTakenDown_ThreatHigh_High_IG_001_SomeoneGotKnocked=Jemand wurde niedergeschlagen!
 PU_GENOUTLAW2_M_CB_Relaxed_NoticeFriendlyTakenDown_ThreatHigh_High_IG_002_TakingInjuries=Verletzungen einstecken!
```

Comment:
"Verletzungen Einstecken!" Klingt irgendwie falsch, vielleicht wäre "Bin verletzt!" oder "Wurde getroffen!" eine bessere übersetzung?

Reply:
Changed to "Mich hat´s erwischt!"